app/assets/javascripts/payment.js

```diff
@@ -0,0 +1,23 @@
+
+jQuery(function($) {
+  $('#payment-form').submit(function(event) {
+    var $form = $(this);
+    $form.find('button').prop('disabled', true);
+    Stripe.card.createToken($form, stripeResponseHandler);
+    return false;
+  });
+});
+
+function stripeResponseHandler(status, response) {
+  var $form = $('#payment-form');
+
+  if (response.error) {
+    $form.find('.payment-errors').text(response.error.message);
+    $form.find('.payment-errors').css('display', 'block');
```

Comment:
Hi Tomas, 
I have add a payment-errors class in my sass file, but how can I add this class instead of manipulate it? Can you give me some reference, thank you~